include/ck/wrapper/utils/layout_utils.hpp

```diff
@@ -300,7 +314,7 @@ __host__ __device__ constexpr auto depth(const T& elem)
  * \return Requsted strides.
  */
 template <typename Shape, typename Strides>
-__host__ __device__ constexpr auto stride(const Layout<Shape, Strides>& layout)
+__host__ __device__ constexpr auto& stride(const Layout<Shape, Strides>& layout)
```

Comment:
`const auto&`. Here it's not as easy to see that the reference is indeed const, so if you want to stick to `auto`, maybe you could be more explicit about its constness?